src/pages/Ahnsangjun/Login/LoginAsj.js

```diff
@@ -1,6 +1,82 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './LoginAsj.scss';
+import { useNavigate } from 'react-router-dom';
 
 export default function LoginAsj() {
-  return <div>Login_asj</div>;
+  const [id, setId] = useState('');
+  const [password, setPassword] = useState('');
+
+  const navigate = useNavigate();
+
+  const idValue = e => {
+    setId(e.target.value);
+  };
+
+  const pwValue = e => {
+    setPassword(e.target.value);
+  };
+
+  const [getData, setGetData] = useState('');
+  console.log(getData);
+
+  const loginSuccess = e => {
+    fetch('http://10.58.52.147:3000/auth/signin', {
+      method: 'POST',
+      headers: { 'Content-type': 'application/json;charset=utf-8' },
+      body: JSON.stringify({ email: id, password: password }),
+    })
+      .then(response => response.json()) //요청
+      .then(data => {
+        setGetData(data); //응답
+        localStorage.setItem('idKey', JSON.stringify(data));
+      });
+    navigate('/mainasj');
+  };
+
+  let loginIdPw = id.indexOf('@') !== -1 && password.length >= '5';
```

Comment:
- 재할당 되는 변수가 아니라면 const로 선언
- T/F를 처리하는 로직이라면, indexOf 대신 includes 사용
- length를 활용한 비교에서 string 대신 number 형식 사용
```suggestion
  const isInputsValid = id.includes('@') && password.length >= 5;
```